version

```diff
@@ -1,2 +1,4 @@
 REPO_VERSION=1
-REPO_NAME=modular-spark
+REPO_NAME="modular-spark"
+BASE_NETWORK_NAME=${REPO_NAME}-${REPO_VERSION}-NETWORK
```

Comment:
Very minor comment.  For line 3 and line 4, we could put a 'v' or 'V' before the version number. ${REPO_NAME-v${REPO_VERSION}-NETWORK.  But this is easy to change later.